index.bs

```diff
@@ -162,9 +160,6 @@ When an item is of type <dfn export for="AV1 Image Item Type">av01</dfn>, it is
     - The content of an [=AV1 Image Item=] is called the <dfn export>AV1 Image Item Data</dfn> and shall obey the following constraints:
         - <assert>The [=AV1 Image Item Data=] shall be identical to the content of an [=AV1 Sample=] marked as 'sync', as defined in [[!AV1-ISOBMFF]].</assert>
         - <assert>The [=AV1 Image Item Data=] shall have exactly one [=Sequence Header OBU=].</assert>
-        - If the [=AV1 Image Item Data=] consists of a single frame (i.e. when using a single layer),
-            - <assert>It should have its <code>[=still_picture=]</code> flag set to 1.</assert>
-            - <assert>It should have its <code>[=reduced_still_picture_header=]</code> flag set to 1.</assert>
 
```

Comment:
Per discussion, we could add something like the following here:

```
NOTE: 
File writers may want to set the <code>[=still_picture=]</code> and 
<code>[=reduced_still_picture_header=]</code> flags to 1 when possible in the [=AV1 Image Item Data=] 
so that header overhead is minimized.
```

Reply:
Leo: Thanks for suggesting the text for the note. The note needs to mention `[=Sequence Header OBU=]` and make it clear that <code>[=still_picture=]</code> and  <code>[=reduced_still_picture_header=]</code> are syntax elements in the `[=Sequence Header OBU=]`. Another possible improvement is to clarify "header overhead" refers to the overhead in the AV1 sequence header and frame header. Perhaps "AV1 header overhead" would suffice.

Reply:
Sounds good to me!
@y-guyon, what do you think?

Reply:
Done.